lib/fathead/mdn_css/parse.pl

```diff
@@ -227,14 +242,14 @@ sub create_abstract {
     }
     $initial_value =~ s/\r?\n+/\\n/g if $initial_value;
     $code = _clean_code($code) if $code;
-    my $out;
-    $out .= "<p>$description</p>"           if $description;
-    $out .= "<p>$initial_value</p>"         if $initial_value;
-    $out .= "<pre><code>$code</code></pre>" if $code;
-    return $out;
+    my $out = Mojo::DOM->new('<div class="prog__container"></div>');
```

Comment:
@hchienjo we just found out that using a `<div>` here is going to cause unseen problems -- can please you switch it to a `<section>` instead?

I'm going to update the docs shortly!

Thanks 👍